docs/product/stats/index.mdx

```diff
@@ -10,11 +10,87 @@ The "Usage" tab shows your organization’s event and attachments usage, while a
+
+#### Accepted
+
+Events and attachments that were successfully processed and stored.
+
+#### Discarded
+
+Events and attachments discarded on the client side due to the sampling configuration in the SDK or other factors.
+
+#### Filtered
+
+| Reason                                                                              | Meaning                                                                            |
+| ----------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
+| [Browser Extensions](/concepts/data-management/filtering/#browser-extension-errors) | Filtered by browser extension.                                                     |
+| Denied Name                                                                         | Filtered due to name being denied.                                                 |
+| Disabled Namespace                                                                  | Filtered due to the namespace being disabled.                                      |
+| Error Message                                                                       | Filtered based on specific error message.                                          |
+| Filtered Transactions                                                               | Filtered because it was a call to a filtered transaction.                          |
+| Ip Address                                                                          | Filtered by IP address.                                                            |
+| Invalid CSP                                                                         | Filtered due to an invalid Content Security Policy (CSP) policy.                   |
+| [Legacy Browsers](/concepts/data-management/filtering/#legacy-browser-filters)      | Filtered out certain legacy versions of browsers that are known to cause problems. |
+| Localhost                                                                           | Filtered due to localhost restriction.                                             |
+| Release Version                                                                     | Filtered by release name (version).                                                |
+| [Web Crawlers](/concepts/data-management/filtering/#web-crawler-errors)             | Identified as known web crawler.                                                   |
+| ? (Others)                                                                          | Filtered due to other generic filtering rules not specified above.                 |
+
+#### Dropped
+
+Events and attachments discarded by the system due to rate limits, quotas, spike protection, [size limits](/concepts/data-management/size-limits), or invalid data. You can see the common reasons below:
```

Comment:
This combines the two outcomes `RATE_LIMITED` and `INVALID`. Do you really want to mix those?

Reply:
yes, because we are combining them in the code https://github.com/getsentry/sentry/blob/566f4a0a04d9a9dd6f1a56d13808ea715b913e8b/static/app/views/organizationStats/usageStatsOrg.tsx#L391-L396